content/conditions/flu/main-content.md

```diff
@@ -0,0 +1,84 @@
+::: reveal Telling the difference between cold and flu 
+  Cold and flu symptoms are similar but flu tends to be more severe. 
+  
+  {.panel}
+  Flu 
+  Cold 
+  Appears quickly within a few hours
+  Appears gradually 
+  Affects more than just your nose and throat
+  Affects mainly your nose and throat 
+  Makes you feel exhausted and too unwell to carry on as normal
+  Makes you feel unwell but you’re ok to carry on as normal – for example, go to work 
+:::
+
+## How to treat flu yourself
+
+To help you get better more quickly:
+
+- rest and sleep
+- keep warm
+- take paracetamol or ibuprofen to lower your temperature and treat aches and pains
+- drink plenty of water to [avoid dehydration](http://www.nhs.uk/Conditions/dehydration/Pages/Introduction.aspx)
+  (your pee should be light yellow or clear)
+
+
+## Your pharmacist can help with flu 
+
+Your pharmacist can give treatment advice and recommend flu remedies. 
+
+Be careful not to use them if you’re taking paracetamol and ibuprofen tablets as it’s easy to take more than the recommended dose. 
+
+Speak to your pharmacist before giving medicines to children. 
+
+[Find your nearest pharmacy]({{ findersBaseUrl }}finders/find-help)
+
+!!! attention
+  ## See your GP if
+  
+  - your symptoms don’t improve after 7 days
+  - your temperature is 38C or above
+  - you’re worried about your child’s symptoms 
+  - you're 65 or over
+  - you're pregnant 
+  - you have a long-term medical condition – for example, diabetes or a heart, lung, kidney or neurological disease
+  - you have a weakened immune system – for example, because of chemotherapy or HIV
+!!!
+
+### Antibiotics 
+
+GPs don’t usually recommend antibiotics for flu because they won’t relieve your symptoms or speed up your recovery.
+
+!!! severe
+  ## Call 999 or go to A&E if you: 
+  
+  - develop chest pain
+  - have shortness of breath 
+  - have difficulty breathing  
+  - start coughing up blood
+!!!
+
+
+## How to avoid spreading the flu 
+
+Flu is very infectious and easily spread to other people. You’re more likely to give it to others in the first 5 days.
+
+Flu is spread by germs from coughs and sneezes which can live on hands and surfaces for 24 hours.
+
+To reduce the risk of spreading flu:
+
+- wash your hands often with warm water and soap
+- use tissues to trap germs when you cough or sneeze
+- bin used tissues as quickly as possible
+
+### How to prevent flu
+
+The flu vaccine reduces the risk of catching flu as well as spreading it to others. 
+
+It’s more effective to get the vaccine before the start of the flu season (December to March).
+
+[Find out if you’re eligible for the free NHS flu  vaccine](http://www.nhs.uk/Conditions/vaccinations/Pages/who-should-have-flu-vaccine.aspx).
```

Comment:
Double space in link wording. Should also include `.` in link if not separating the link. Otherwise some screen readers will read out the punctuation separately after the link.